scripts/zones/RoMaeve/npcs/qm3.lua

```diff
@@ -0,0 +1,30 @@
+-----------------------------------
+-- Area: Ro'Maeve
+--  NPC: qm3 (Moongate Pass QM)
+-- !pos -277.651,-3.765,-17.895 122 and many <pos>
+-----------------------------------
+local ID = require("scripts/zones/RoMaeve/IDs")
+require("scripts/globals/npc_util")
+require("scripts/globals/settings")
+require("scripts/globals/keyitems")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    if player:hasKeyItem(dsp.ki.MOONGATE_PASS) then
+        player:messageSpecial(ID.text.NOTHING_OUT_OF_ORDINARY)
+    else
+        npcUtil.giveKeyItem(player, dsp.ki.MOONGATE_PASS)
+        npc:hideNPC(3000)
+        local newPosition = npcUtil.pickNewPosition(npc:getID(), ID.npc.MOONGATE_PASS_QM_POS, allowCurrentPosition)
```

Comment:
`allowCurrentPosition` not defined

Reply:
ah crap that was meant to be true! thank you for catch @ibm2431  😸